lib/Reader/readerir.cpp

```diff
@@ -5308,7 +5309,8 @@ void GenIR::nop() {
     // We may need to pick something else that survives lowering.
     Value *DoNothing = Intrinsic::getDeclaration(JitContext->CurrentModule,
                                                  Intrinsic::donothing);
-    LLVMBuilder->CreateCall(DoNothing);
+    ArrayRef<Value *> Args;
```

Comment:
I'm a little surprised that this works. I would have expected something like `Value *Args[] = {};`.